rsrcdump/structtemplate.py

```diff
@@ -90,7 +95,7 @@ def unpack_record(self, data: bytes, offset: int) -> Any:
         values = struct.unpack_from(self.format, data, offset)
         return self.tag_values(values)
 
-    def tag_values(self, values: tuple):
+    def tag_values(self, values: tuple[str | int, ...]) -> dict[str, str | int] | str | int | tuple[str | int, ...]:
```

Comment:
This is misleading – `values` is a series of arbitrary struct fields which can be any type.

Reply:
Updated in e907f63